helm-chart/renku-notebooks/templates/configmap.yaml

```diff
@@ -9,8 +9,10 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
-  server_options.json: |
-    {{ toJson .Values.serverOptions }}
+  server_options_ui.json: |
```

Comment:
Could we also continue using `serverOptions` (instead of `serverOptionsUI`) for what's advertised to clients, and `serverDefaults` (instead of `serverOptions`) for the new defaults? This would avoid the breaking change.